docs/pages/gettingstarted/extended/peer.md

```diff
@@ -3,74 +3,61 @@ title: Enable peering
+Once Liqo is installed on your clusters, you can start establishing new *peerings*. 
+
 
 ## Create the desired multi-cluster architecture
 
 From now on, the *cluster-1* is the *home-cluster*. 
 You can enable the *home-cluster* to peer with the other 2 clusters.
 
-### Enable peerings
-
-Using *kubectl*, you can obtain the list of foreign clusters discovered by the *home-cluster*:
+### Enable peering
 
-```bash
-export KUBECONFIG=$KUBECONFIG_1
-kubectl get foreignclusters
 ```
-
-There should be two *ForeignCluster* resources in that state:
-
-```bash
-NAME                                   OUTGOING PEERING PHASE   INCOMING PEERING PHASE   NETWORKING STATUS   AUTHENTICATION STATUS
-b07938e3-d241-460c-a77b-e286c0f733c7   None                     None                     None                Established             
-b38f5c32-a877-4f82-8bde-2fd0c5c8f862   None                     None                     None                Established             
+export KUBECONFIG=$KUBECONFIG_2
+liqoctl generate-command
 ```
 
-{{% notice note %}}
-When discovered using LAN discovery, the ForeignCluster object name is the cluster-id of the corresponding remote cluster.
-{{% /notice %}}
-
-To enable an unidirectional peering it is sufficient to edit a field of the ForeignCluster resource:
+You will obtain an output like the following:
 
 ```bash
-kubectl patch foreignclusters <your-ForeignCluster-name> \
---patch '{"spec":{"outgoingPeeringEnabled":"Yes"}}' \
---type 'merge'
+liqoctl add cluster cluster-2 --auth-url https://172.18.0.5:32714 \ 
+    --id 3623b0bd-3c32-4dec-994b-fc80d9d0d91d \
+    --token b13b6932ee6fd890a1abe212dc21253aa6d74565fead54
 ```
 
-Repeat the same operation for the other resource.
-
-After the previous patches to ForeignCluster resources, you should obtain:
+This output represents the command to use to enable an outgoing peering from another cluster to cluser-2.
+Therefore, you can take the output obtained and launch it aftering having sourced the KUBECONFIG_1:
 
-```bash
+```
 export KUBECONFIG=$KUBECONFIG_1
-kubectl get foreignclusters
+liqoctl add ... # the output you obtained from the liqoctl generate-command
 ```
 
-```bash
-NAME                                   OUTGOING PEERING PHASE   INCOMING PEERING PHASE   NETWORKING STATUS   AUTHENTICATION STATUS
-b07938e3-d241-460c-a77b-e286c0f733c7   Established              None                     None                Established          
-b38f5c32-a877-4f82-8bde-2fd0c5c8f862   Established              None                     None                Established          
-```
+You can do the same with cluster-3:
 
-Only the "*OUTGOING PEERING PHASE*" is set to Established, so only the *home-cluster* can use the resources of the other two clusters:
+```
+export KUBECONFIG=$KUBECONFIG_3
```

Comment:
Shouldn't it be $KUBECONFIG_1 instead of $KUBECONFIG_3? The outgoing peering should from home-cluster to cluster-3 if I understand well